cdisc_rules_engine/rules_engine.py

```diff
@@ -66,10 +66,14 @@ def __init__(
         self.ct_package = kwargs.get("ct_package")
         self.meddra_path: str = kwargs.get("meddra_path")
         self.whodrug_path: str = kwargs.get("whodrug_path")
+        self.validate_xml: str = kwargs.get("validate_xml")
 
     def get_schema(self):
         return export_rule_data(DatasetVariable, COREActions)
 
+    def is_validate_xml(self):
```

Comment:
Does this need to be defined here and in the base_define_reader class? Maybe it should be a utility method